draft-ietf-core-coap-tcp-tls.md

```diff
@@ -1267,7 +1276,7 @@ established with {{-urireg}}.
 
 ### coap+ws

```

Comment:
The text is mostly with MS-DOS line endings; however a few lines have UNIX line endings.  I can't easily edit mixed line endings, so whenever I edit something, the minority line endings will revert to majority line endings.  It is generally painful to work on text with MS-DOS line endings on Github; I would recommend reverting to the usual UNIX line endings when we are done with -03.